tests/test_user_data_parser.py

```diff
@@ -65,6 +65,9 @@ def test_read_files__success(self):
         parser.read_data(os.path.join(VALID_USER_DATA_DIRECTORY,
                                       'acls.yaml'))
 
+        parser.read_data(os.path.join(VALID_USER_DATA_DIRECTORY,
+                                      'empty.yml'))
```

Comment:
Could you put some comments in the file?  The real case that Ben has is that the samples are all commented out.

Reply:
Comments added